### PR TITLE
Updated transform.py

### PR DESCRIPTION
**Pull request by sganes21**
- Added initial commit to transform, to parse .json and convert to panda dataframe
- Assumes that the 'data' key in the API response always contains a list of dictionaries, each representing a stock entry.
- Have not added error handling
- Have not tested yet

### CODE REVIEW

File: steps/transform.py [hunk 1]
```diff
@@ -1,13 +1,15 @@
...
+import pandas as pd
+
 
-def backward(str):
-    return str[::-1]
 
 def lambda_handler(event, context):
-    payload = json.loads(event['body'])
+    transform_response = lambda response: pd.DataFrame(json.loads(response)['data'])
+    df = transform_response()
```
Comment (haranku16) [inline steps/transform.py]: We'll need to pass event to this function

File: steps/transform.py [hunk 2]
```diff
@@ -1,13 +1,15 @@
...
+
     return {
         'statusCode': 200,
-        'body': json.dumps({
-            'data': [backward(data['name']) for data in payload['data']]
-        })
+        'body': json.dumps('Lambda function executed successfully')
```
Comment (haranku16) [inline steps/transform.py]: Instead of json.dumps, can you pass df.to_csv() as the body so that the CSV text gets passed to the next function?

**Review by haranku16 — CHANGES_REQUESTED** [review]

**Review by sganes21 — COMMENTED** [review]
-passed event to transform_response function
-pass the entire CSV content as a string in the body of the Lambda function's response.
-added optional headers to the response to specify the content type as CSV and suggest a filename for download.

**Review by haranku16 — APPROVED** [review]
Looks good, thanks!

**Pull request merged**